src/modules/components/proposal/proposalActions/proposalActionsAction/proposalActionsAction.tsx

```diff
@@ -70,7 +71,19 @@ export const ProposalActionsAction: React.FC<IProposalActionsActionProps> = (pro
                     <ProposalActionsActionVerification action={action} />
                 </div>
             </Accordion.ItemHeader>
-            <Accordion.ItemContent>{ActionComponent}</Accordion.ItemContent>
+            <Accordion.ItemContent>
+                {action.value != null && action.value !== '0' && (
+                    <AlertCard
+                        variant="critical"
+                        message={copy.proposalActionsAction.nativeSendAlert}
+                        description={copy.proposalActionsAction.nativeSendDescription(
+                            formatUnits(BigInt(action.value), 18),
+                        )}
+                        className="mb-6 md:mb-8"
```

Comment:
Can we use flex with gap instead of margin bottom?

Reply:
Radix doesn't like it if we pass to the Item.Content... could we wrap with a div instead?

Reply:
@thekidnamedkd Yes sure 👍🏻